library/core/registry/src/testmod/resources/quilt.mod.json

```diff
@@ -27,5 +27,6 @@
       "quilt_registry",
       "quilt_testing"
     ]
-  }
+  },
+  "quilt:mod_protocol": 2
```

Comment:
The standard of custom objects is that they are a mod ID, and inside of it, there would be the relevant stuff, so, it would be better to have a `quilt_registry` object with a `mod_protocol` entry inside
This is only my opinion though, I'd like another person to look at it

Reply:
Yeah, this should be an object.

Reply:
Changed